Core/App/AppInstaller.php

```diff
@@ -70,11 +70,14 @@ public function __construct()
 
         if ($installed) {
             header('Location: ' . $this->getUri());
-        } else {
-            $this->render();
         }
-    }
 
+        if ($this->request->getBasePath() !== $this->getUri()) {
+            header('Location: ' . $this->request->getBasePath());
```

Comment:
Maybe is more appropiate use the symfony way or the framework with a [RedirectReponse](https://api.symfony.com/3.4/Symfony/Component/HttpFoundation/RedirectResponse.html) object

Reply:
Yes, but in this case, I only replace uneeded elses.